Startup error on Alfresco 5.2.g with enabled serial optimisations

## BUG

With aldica 1.0.1 deployed on an Alfresco 5.2.g instance, there is a failure during startup relating to a Hibernate exception about a connection proxy being unusable after transaction completion. This occurs during the initialisation of the file servers subsystem and/or web script registry initialisation, and the error disappears when `aldica.core.binary.optimisation.enabled=false` is set.
Though Alfresco 5.2.g is not officially supported / tested, we do claim it *should* be generally compatible...

### Expected behavior

Startup of Alfresco 5.2.g succeeds in the default configuration of Aldica, or a documented 5.2.g specific configuration.

### Actual behavior

Startup of Alfresco 5.2.g fails with an error that is caused by a `org.hibernate.HibernateException`, specifically with the error message `connnection proxy not usable after transaction completion`

### Steps to reproduce the behavior

Install aldica 1.0.1 on Alfresco 5.2.g and start the system

### Additional details (analysis so far, log statements, references, etc.)

```
[ALFRESCO] 2020-09-21 15:04:06,236  INFO  [management.subsystems.ChildApplicationContextFactory] [localhost-startStop-1] Starting 'fileServers' subsystem, ID: [fileServers, default]
 [ALFRESCO] 2020-09-21 15:04:06,457  INFO  [management.subsystems.ChildApplicationContextFactory] [localhost-startStop-1] Starting 'sysAdmin' subsystem, ID: [sysAdmin, default]
 [ALFRESCO] 2020-09-21 15:04:06,472  INFO  [management.subsystems.ChildApplicationContextFactory] [localhost-startStop-1] Startup of 'sysAdmin' subsystem, ID: [sysAdmin, default] complete
 [ALFRESCO] 2020-09-21 15:04:06,483  ERROR [hibernate.jdbc.BorrowedConnectionProxy] [localhost-startStop-1] Unusable connection proxy encountered: org.hibernate.jdbc.BorrowedConnectionProxy@2b8f820f
 [ALFRESCO] 2020-09-21 15:04:06,484  ERROR [hibernate.jdbc.BorrowedConnectionProxy] [localhost-startStop-1] Logging of call stack is enforced and will affect performance.
 [ALFRESCO] 2020-09-21 15:04:06,493  WARN  [management.subsystems.ChildApplicationContextFactory] [localhost-startStop-1] Startup of 'fileServers' subsystem, ID: [fileServers, default] failed
 org.alfresco.error.AlfrescoRuntimeException: 08210018 File server configuration error, nested exception is org.apache.ibatis.executor.ExecutorException: Error preparing statement.  Cause: org.hibernate.HibernateException: connnection proxy not usable after transaction completion
	at org.alfresco.filesys.AbstractServerConfigurationBean.init(AbstractServerConfigurationBean.java:449)
	at org.alfresco.filesys.AbstractServerConfigurationBean.onApplicationEvent(AbstractServerConfigurationBean.java:790)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:96)
	at org.alfresco.repo.management.subsystems.ChildApplicationContextFactory$ChildApplicationContext.publishEvent(ChildApplicationContextFactory.java:569)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:954)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:482)
	at org.alfresco.repo.management.subsystems.ChildApplicationContextFactory$ApplicationContextState.start(ChildApplicationContextFactory.java:824)
	at org.alfresco.repo.management.subsystems.AbstractPropertyBackedBean.start(AbstractPropertyBackedBean.java:1098)
	at org.alfresco.repo.management.subsystems.AbstractPropertyBackedBean.onApplicationEvent(AbstractPropertyBackedBean.java:637)
	at org.alfresco.repo.management.SafeApplicationEventMulticaster.multicastEventInternal(SafeApplicationEventMulticaster.java:214)
	at org.alfresco.repo.management.SafeApplicationEventMulticaster.multicastEvent(SafeApplicationEventMulticaster.java:185)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:334)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:954)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:482)
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:410)
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:306)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:112)
	at org.alfresco.web.app.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:70)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:5136)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5662)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:1007)
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:983)
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:639)
	at org.apache.catalina.startup.HostConfig.deployDirectory(HostConfig.java:1296)
	at org.apache.catalina.startup.HostConfig$DeployDirectory.run(HostConfig.java:2038)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.executor.ExecutorException: Error preparing statement.  Cause: org.hibernate.HibernateException: connnection proxy not usable after transaction completion
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:76)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:421)
	at com.sun.proxy.$Proxy16.selectList(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.selectList(SqlSessionTemplate.java:214)
	at org.alfresco.repo.domain.node.ibatis.NodeDAOImpl.selectChildAssocs(NodeDAOImpl.java:1219)
	at org.alfresco.repo.domain.node.AbstractNodeDAOImpl.getChildAssocs(AbstractNodeDAOImpl.java:3571)
	at org.alfresco.repo.node.db.DbNodeServiceImpl.getChildAssocs_aroundBody76(DbNodeServiceImpl.java:1931)
	at org.alfresco.repo.node.db.DbNodeServiceImpl$AjcClosure77.run(DbNodeServiceImpl.java:1)
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)
	at org.alfresco.traitextender.RouteExtensions.intercept(RouteExtensions.java:100)
	at org.alfresco.repo.node.db.DbNodeServiceImpl.getChildAssocs(DbNodeServiceImpl.java:1881)
	at org.alfresco.repo.node.db.DbNodeServiceImpl.getChildAssocs_aroundBody74(DbNodeServiceImpl.java:1874)
	at org.alfresco.repo.node.db.DbNodeServiceImpl$AjcClosure75.run(DbNodeServiceImpl.java:1)
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)
	at org.alfresco.traitextender.RouteExtensions.intercept(RouteExtensions.java:100)
	at org.alfresco.repo.node.db.DbNodeServiceImpl.getChildAssocs(DbNodeServiceImpl.java:1868)
	at org.alfresco.repo.node.db.DbNodeServiceImpl.getChildAssocs_aroundBody72(DbNodeServiceImpl.java:1861)
	at org.alfresco.repo.node.db.DbNodeServiceImpl$AjcClosure73.run(DbNodeServiceImpl.java:1)
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)
	at org.alfresco.traitextender.RouteExtensions.intercept(RouteExtensions.java:100)
	at org.alfresco.repo.node.db.DbNodeServiceImpl.getChildAssocs(DbNodeServiceImpl.java:1859)
	at sun.reflect.GeneratedMethodAccessor188.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:183)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
	at org.alfresco.repo.lock.mem.LockableAspectInterceptor.invoke(LockableAspectInterceptor.java:244)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:204)
	at com.sun.proxy.$Proxy27.getChildAssocs(Unknown Source)
	at sun.reflect.GeneratedMethodAccessor188.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:183)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
	at org.alfresco.repo.tenant.MultiTNodeServiceInterceptor.invoke(MultiTNodeServiceInterceptor.java:111)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:204)
	at com.sun.proxy.$Proxy27.getChildAssocs(Unknown Source)
	at sun.reflect.GeneratedMethodAccessor188.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.alfresco.repo.service.StoreRedirectorProxyFactory$RedirectorInvocationHandler.invoke(StoreRedirectorProxyFactory.java:231)
	at com.sun.proxy.$Proxy50.getChildAssocs(Unknown Source)
	at org.alfresco.repo.search.DocumentNavigator.getChildAxisIterator(DocumentNavigator.java:388)
	at org.jaxen.expr.iter.IterableChildAxis.namedAccessIterator(IterableChildAxis.java:98)
	at org.jaxen.expr.DefaultNameStep.evaluate(DefaultNameStep.java:180)
	at org.jaxen.expr.DefaultLocationPath.evaluate(DefaultLocationPath.java:140)
	at org.jaxen.expr.DefaultAbsoluteLocationPath.evaluate(DefaultAbsoluteLocationPath.java:113)
	at org.jaxen.expr.DefaultXPathExpr.asList(DefaultXPathExpr.java:102)
	at org.jaxen.BaseXPath.selectNodesForContext(BaseXPath.java:677)
	at org.jaxen.BaseXPath.selectNodes(BaseXPath.java:216)
	at org.alfresco.repo.search.NodeServiceXPath.selectNodes(NodeServiceXPath.java:193)
	at org.alfresco.repo.search.impl.NodeSearcher.selectNodes(NodeSearcher.java:96)
	at org.alfresco.repo.search.impl.SearchServiceSubSystemDelegator.selectNodes(SearchServiceSubSystemDelegator.java:167)
	at org.alfresco.repo.search.impl.SearchServiceSubSystemDelegator.selectNodes(SearchServiceSubSystemDelegator.java:149)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:183)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
	at net.sf.acegisecurity.intercept.method.aopalliance.MethodSecurityInterceptor.invoke(MethodSecurityInterceptor.java:80)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.alfresco.repo.security.permissions.impl.ExceptionTranslatorMethodInterceptor.invoke(ExceptionTranslatorMethodInterceptor.java:53)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.alfresco.repo.audit.AuditMethodInterceptor.invoke(AuditMethodInterceptor.java:166)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:96)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:260)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:94)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:204)
	at com.sun.proxy.$Proxy30.selectNodes(Unknown Source)
	at org.alfresco.filesys.repo.ContentDiskDriver2$1.doWork(ContentDiskDriver2.java:484)
	at org.alfresco.filesys.repo.ContentDiskDriver2$1.doWork(ContentDiskDriver2.java:1)
	at org.alfresco.repo.security.authentication.AuthenticationUtil.runAs(AuthenticationUtil.java:555)
	at org.alfresco.filesys.repo.ContentDiskDriver2.registerContext(ContentDiskDriver2.java:556)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:198)
	at com.sun.proxy.$Proxy305.registerContext(Unknown Source)
	at org.alfresco.filesys.repo.NonTransactionalRuleContentDiskDriver.registerContext(NonTransactionalRuleContentDiskDriver.java:609)
	at org.alfresco.filesys.repo.LegacyFileStateDriver.registerContext(LegacyFileStateDriver.java:541)
	at org.alfresco.filesys.repo.BufferedContentDiskDriver.registerContext(BufferedContentDiskDriver.java:571)
	at org.alfresco.filesys.config.ServerConfigurationBean.processFilesystemsConfig(ServerConfigurationBean.java:1638)
	at org.alfresco.filesys.AbstractServerConfigurationBean.init(AbstractServerConfigurationBean.java:444)
	... 30 more
Caused by: org.apache.ibatis.executor.ExecutorException: Error preparing statement.  Cause: org.hibernate.HibernateException: connnection proxy not usable after transaction completion
	at org.apache.ibatis.executor.statement.BaseStatementHandler.prepare(BaseStatementHandler.java:97)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.prepare(RoutingStatementHandler.java:58)
	at org.apache.ibatis.executor.SimpleExecutor.prepareStatement(SimpleExecutor.java:76)
	at org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:61)
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:303)
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:154)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:102)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:82)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:120)
	at sun.reflect.GeneratedMethodAccessor159.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:408)
	... 121 more
Caused by: org.hibernate.HibernateException: connnection proxy not usable after transaction completion
	at org.hibernate.jdbc.BorrowedConnectionProxy.invoke(BorrowedConnectionProxy.java:142)
	at com.sun.proxy.$Proxy14.prepareStatement(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.instantiateStatement(PreparedStatementHandler.java:79)
	at org.apache.ibatis.executor.statement.BaseStatementHandler.prepare(BaseStatementHandler.java:88)
	... 133 more
```

### Tell us about your environment

- OpenJDK 1.8 / Tomcat 8.5
- Alfresco 5.2.g
- Order of the Bee Support Tools 1.1.0.0
- JavaScript Console 0.6
- Acosix Alfresco Utility Core 1.2.2
- aldica 1.0.1

Interestingly, the specific stack trace reported shifts depending on yet to be understood circumstances. In my debugging attempts, it consistently is the following:

```
repository_1  | Sep 27 13:17:16 6c1f836f9e21 2020-09-27 13:17:16,190 INFO  [org.alfresco.repo.management.subsystems.ChildApplicationContextFactory] [localhost-startStop-1] Starting 'fileServers' subsystem, ID: [fileServers, default]
repository_1  | Sep 27 13:17:17 6c1f836f9e21 2020-09-27 13:17:16,868 INFO  [org.alfresco.repo.management.subsystems.ChildApplicationContextFactory] [localhost-startStop-1] Starting 'sysAdmin' subsystem, ID: [sysAdmin, default]
repository_1  | Sep 27 13:17:17 6c1f836f9e21 2020-09-27 13:17:16,898 INFO  [org.alfresco.repo.management.subsystems.ChildApplicationContextFactory] [localhost-startStop-1] Startup of 'sysAdmin' subsystem, ID: [sysAdmin, default] complete
repository_1  | Sep 27 13:18:51 6c1f836f9e21 2020-09-27 13:18:51,401 ERROR [org.hibernate.jdbc.BorrowedConnectionProxy] [localhost-startStop-1] Unusable connection proxy encountered: org.hibernate.jdbc.BorrowedConnectionProxy@75f174c5
repository_1  | Sep 27 13:18:51 6c1f836f9e21 2020-09-27 13:18:51,401 ERROR [org.hibernate.jdbc.BorrowedConnectionProxy] [localhost-startStop-1] Logging of call stack is enforced and will affect performance.
repository_1  | Sep 27 13:18:51 6c1f836f9e21 2020-09-27 13:18:51,415 WARN  [org.alfresco.repo.management.subsystems.ChildApplicationContextFactory] [localhost-startStop-1] Startup of 'fileServers' subsystem, ID: [fileServers, default] failed
repository_1  | Sep 27 13:18:51 6c1f836f9e21 org.alfresco.error.AlfrescoRuntimeException: 08270018 File server configuration error, nested exception is org.apache.ibatis.executor.ExecutorException: Error preparing statement.  Cause: org.hibernate.HibernateException: connnection proxy not usable after transaction completion
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.AbstractServerConfigurationBean.init(AbstractServerConfigurationBean.java:449)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.AbstractServerConfigurationBean.onApplicationEvent(AbstractServerConfigurationBean.java:790)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:96)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.management.subsystems.ChildApplicationContextFactory$ChildApplicationContext.publishEvent(ChildApplicationContextFactory.java:569)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:954)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:482)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.management.subsystems.ChildApplicationContextFactory$ApplicationContextState.start(ChildApplicationContextFactory.java:824)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.management.subsystems.AbstractPropertyBackedBean.start(AbstractPropertyBackedBean.java:1098)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.management.subsystems.AbstractPropertyBackedBean.onApplicationEvent(AbstractPropertyBackedBean.java:637)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.management.SafeApplicationEventMulticaster.multicastEventInternal(SafeApplicationEventMulticaster.java:214)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.management.SafeApplicationEventMulticaster.multicastEvent(SafeApplicationEventMulticaster.java:185)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:334)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:954)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:482)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:410)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:306)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:112)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.web.app.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:70)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4689)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5155)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:743)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:719)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:614)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1822)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.Thread.run(Thread.java:748)
repository_1  | Sep 27 13:18:51 6c1f836f9e21 Caused by: org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.executor.ExecutorException: Error preparing statement.  Cause: org.hibernate.HibernateException: connnection proxy not usable after transaction completion
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:76)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:421)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy13.selectOne(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.mybatis.spring.SqlSessionTemplate.selectOne(SqlSessionTemplate.java:166)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.domain.qname.ibatis.QNameDAOImpl.findQNameEntityByNamespaceAndLocalName(QNameDAOImpl.java:109)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.domain.qname.AbstractQNameDAOImpl$QNameCallbackDAO.findByValue(AbstractQNameDAOImpl.java:361)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.domain.qname.AbstractQNameDAOImpl$QNameCallbackDAO.findByValue(AbstractQNameDAOImpl.java:1)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.cache.lookup.EntityLookupCache.getByValue(EntityLookupCache.java:405)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.domain.qname.AbstractQNameDAOImpl.getQName(AbstractQNameDAOImpl.java:254)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.GeneratedMethodAccessor157.invoke(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:198)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy16.getQName(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.domain.node.NodeEntity.getDeleted(NodeEntity.java:227)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.domain.node.AbstractNodeDAOImpl.getNodePair(AbstractNodeDAOImpl.java:1038)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.node.db.DbNodeServiceImpl.getNodePairNotNull(DbNodeServiceImpl.java:194)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.node.db.DbNodeServiceImpl.getPrimaryParent_aroundBody84(DbNodeServiceImpl.java:2046)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.node.db.DbNodeServiceImpl$AjcClosure85.run(DbNodeServiceImpl.java:1)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.traitextender.RouteExtensions.intercept(RouteExtensions.java:100)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.node.db.DbNodeServiceImpl.getPrimaryParent(DbNodeServiceImpl.java:2043)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.GeneratedMethodAccessor260.invoke(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:183)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.lock.mem.LockableAspectInterceptor.invoke(LockableAspectInterceptor.java:244)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:204)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy23.getPrimaryParent(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.GeneratedMethodAccessor260.invoke(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:183)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.tenant.MultiTNodeServiceInterceptor.invoke(MultiTNodeServiceInterceptor.java:111)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:204)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy23.getPrimaryParent(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.GeneratedMethodAccessor260.invoke(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.service.StoreRedirectorProxyFactory$RedirectorInvocationHandler.invoke(StoreRedirectorProxyFactory.java:231)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy45.getPrimaryParent(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.search.impl.NodeSearcher.selectNodes(NodeSearcher.java:96)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.search.impl.SearchServiceSubSystemDelegator.selectNodes(SearchServiceSubSystemDelegator.java:167)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.search.impl.SearchServiceSubSystemDelegator.selectNodes(SearchServiceSubSystemDelegator.java:149)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:183)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at net.sf.acegisecurity.intercept.method.aopalliance.MethodSecurityInterceptor.invoke(MethodSecurityInterceptor.java:80)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.security.permissions.impl.ExceptionTranslatorMethodInterceptor.invoke(ExceptionTranslatorMethodInterceptor.java:53)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.audit.AuditMethodInterceptor.invoke(AuditMethodInterceptor.java:166)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:96)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:260)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:94)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:204)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy26.selectNodes(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.repo.ContentDiskDriver2$1.doWork(ContentDiskDriver2.java:484)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.repo.ContentDiskDriver2$1.doWork(ContentDiskDriver2.java:1)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.security.authentication.AuthenticationUtil.runAs(AuthenticationUtil.java:555)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.repo.ContentDiskDriver2.registerContext(ContentDiskDriver2.java:556)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:198)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy232.registerContext(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.repo.NonTransactionalRuleContentDiskDriver.registerContext(NonTransactionalRuleContentDiskDriver.java:609)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.repo.LegacyFileStateDriver.registerContext(LegacyFileStateDriver.java:541)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.repo.BufferedContentDiskDriver.registerContext(BufferedContentDiskDriver.java:571)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.config.ServerConfigurationBean.processFilesystemsConfig(ServerConfigurationBean.java:1638)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.AbstractServerConfigurationBean.init(AbstractServerConfigurationBean.java:444)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    ... 30 more
repository_1  | Sep 27 13:18:51 6c1f836f9e21 Caused by: org.apache.ibatis.executor.ExecutorException: Error preparing statement.  Cause: org.hibernate.HibernateException: connnection proxy not usable after transaction completion
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.statement.BaseStatementHandler.prepare(BaseStatementHandler.java:97)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.statement.RoutingStatementHandler.prepare(RoutingStatementHandler.java:58)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.SimpleExecutor.prepareStatement(SimpleExecutor.java:76)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:61)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:303)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:154)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:102)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:82)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:120)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:113)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:73)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.GeneratedMethodAccessor140.invoke(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:408)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    ... 114 more
repository_1  | Sep 27 13:18:51 6c1f836f9e21 Caused by: org.hibernate.HibernateException: connnection proxy not usable after transaction completion
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.hibernate.jdbc.BorrowedConnectionProxy.invoke(BorrowedConnectionProxy.java:142)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy11.prepareStatement(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.statement.PreparedStatementHandler.instantiateStatement(PreparedStatementHandler.java:79)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.statement.BaseStatementHandler.prepare(BaseStatementHandler.java:88)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    ... 128 more
repository_1  | Sep 27 13:18:51 6c1f836f9e21 2020-09-27 13:18:51,425 ERROR [org.alfresco.repo.management.subsystems.AbstractPropertyBackedBean] [localhost-startStop-1] Error auto-starting subsystem
repository_1  | Sep 27 13:18:51 6c1f836f9e21 org.alfresco.error.AlfrescoRuntimeException: 08270018 File server configuration error, nested exception is org.apache.ibatis.executor.ExecutorException: Error preparing statement.  Cause: org.hibernate.HibernateException: connnection proxy not usable after transaction completion
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.AbstractServerConfigurationBean.init(AbstractServerConfigurationBean.java:449)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.AbstractServerConfigurationBean.onApplicationEvent(AbstractServerConfigurationBean.java:790)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:96)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.management.subsystems.ChildApplicationContextFactory$ChildApplicationContext.publishEvent(ChildApplicationContextFactory.java:569)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:954)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:482)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.management.subsystems.ChildApplicationContextFactory$ApplicationContextState.start(ChildApplicationContextFactory.java:824)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.management.subsystems.AbstractPropertyBackedBean.start(AbstractPropertyBackedBean.java:1098)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.management.subsystems.AbstractPropertyBackedBean.onApplicationEvent(AbstractPropertyBackedBean.java:637)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.management.SafeApplicationEventMulticaster.multicastEventInternal(SafeApplicationEventMulticaster.java:214)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.management.SafeApplicationEventMulticaster.multicastEvent(SafeApplicationEventMulticaster.java:185)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:334)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:954)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:482)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:410)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:306)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:112)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.web.app.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:70)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4689)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5155)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:743)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:719)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:614)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1822)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.Thread.run(Thread.java:748)
repository_1  | Sep 27 13:18:51 6c1f836f9e21 Caused by: org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.executor.ExecutorException: Error preparing statement.  Cause: org.hibernate.HibernateException: connnection proxy not usable after transaction completion
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:76)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:421)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy13.selectOne(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.mybatis.spring.SqlSessionTemplate.selectOne(SqlSessionTemplate.java:166)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.domain.qname.ibatis.QNameDAOImpl.findQNameEntityByNamespaceAndLocalName(QNameDAOImpl.java:109)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.domain.qname.AbstractQNameDAOImpl$QNameCallbackDAO.findByValue(AbstractQNameDAOImpl.java:361)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.domain.qname.AbstractQNameDAOImpl$QNameCallbackDAO.findByValue(AbstractQNameDAOImpl.java:1)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.cache.lookup.EntityLookupCache.getByValue(EntityLookupCache.java:405)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.domain.qname.AbstractQNameDAOImpl.getQName(AbstractQNameDAOImpl.java:254)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.GeneratedMethodAccessor157.invoke(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:198)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy16.getQName(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.domain.node.NodeEntity.getDeleted(NodeEntity.java:227)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.domain.node.AbstractNodeDAOImpl.getNodePair(AbstractNodeDAOImpl.java:1038)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.node.db.DbNodeServiceImpl.getNodePairNotNull(DbNodeServiceImpl.java:194)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.node.db.DbNodeServiceImpl.getPrimaryParent_aroundBody84(DbNodeServiceImpl.java:2046)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.node.db.DbNodeServiceImpl$AjcClosure85.run(DbNodeServiceImpl.java:1)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.traitextender.RouteExtensions.intercept(RouteExtensions.java:100)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.node.db.DbNodeServiceImpl.getPrimaryParent(DbNodeServiceImpl.java:2043)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.GeneratedMethodAccessor260.invoke(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:183)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.lock.mem.LockableAspectInterceptor.invoke(LockableAspectInterceptor.java:244)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:204)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy23.getPrimaryParent(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.GeneratedMethodAccessor260.invoke(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:183)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.tenant.MultiTNodeServiceInterceptor.invoke(MultiTNodeServiceInterceptor.java:111)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:204)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy23.getPrimaryParent(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.GeneratedMethodAccessor260.invoke(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.service.StoreRedirectorProxyFactory$RedirectorInvocationHandler.invoke(StoreRedirectorProxyFactory.java:231)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy45.getPrimaryParent(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.search.impl.NodeSearcher.selectNodes(NodeSearcher.java:96)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.search.impl.SearchServiceSubSystemDelegator.selectNodes(SearchServiceSubSystemDelegator.java:167)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.search.impl.SearchServiceSubSystemDelegator.selectNodes(SearchServiceSubSystemDelegator.java:149)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:183)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at net.sf.acegisecurity.intercept.method.aopalliance.MethodSecurityInterceptor.invoke(MethodSecurityInterceptor.java:80)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.security.permissions.impl.ExceptionTranslatorMethodInterceptor.invoke(ExceptionTranslatorMethodInterceptor.java:53)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.audit.AuditMethodInterceptor.invoke(AuditMethodInterceptor.java:166)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:96)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:260)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:94)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:204)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy26.selectNodes(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.repo.ContentDiskDriver2$1.doWork(ContentDiskDriver2.java:484)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.repo.ContentDiskDriver2$1.doWork(ContentDiskDriver2.java:1)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.repo.security.authentication.AuthenticationUtil.runAs(AuthenticationUtil.java:555)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.repo.ContentDiskDriver2.registerContext(ContentDiskDriver2.java:556)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:198)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy232.registerContext(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.repo.NonTransactionalRuleContentDiskDriver.registerContext(NonTransactionalRuleContentDiskDriver.java:609)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.repo.LegacyFileStateDriver.registerContext(LegacyFileStateDriver.java:541)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.repo.BufferedContentDiskDriver.registerContext(BufferedContentDiskDriver.java:571)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.config.ServerConfigurationBean.processFilesystemsConfig(ServerConfigurationBean.java:1638)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.alfresco.filesys.AbstractServerConfigurationBean.init(AbstractServerConfigurationBean.java:444)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    ... 30 more
repository_1  | Sep 27 13:18:51 6c1f836f9e21 Caused by: org.apache.ibatis.executor.ExecutorException: Error preparing statement.  Cause: org.hibernate.HibernateException: connnection proxy not usable after transaction completion
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.statement.BaseStatementHandler.prepare(BaseStatementHandler.java:97)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.statement.RoutingStatementHandler.prepare(RoutingStatementHandler.java:58)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.SimpleExecutor.prepareStatement(SimpleExecutor.java:76)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:61)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:303)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:154)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:102)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:82)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:120)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:113)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:73)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.GeneratedMethodAccessor140.invoke(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at java.lang.reflect.Method.invoke(Method.java:498)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:408)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    ... 114 more
repository_1  | Sep 27 13:18:51 6c1f836f9e21 Caused by: org.hibernate.HibernateException: connnection proxy not usable after transaction completion
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.hibernate.jdbc.BorrowedConnectionProxy.invoke(BorrowedConnectionProxy.java:142)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at com.sun.proxy.$Proxy11.prepareStatement(Unknown Source)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.statement.PreparedStatementHandler.instantiateStatement(PreparedStatementHandler.java:79)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    at org.apache.ibatis.executor.statement.BaseStatementHandler.prepare(BaseStatementHandler.java:88)
repository_1  | Sep 27 13:18:51 6c1f836f9e21    ... 128 more
```

--- a/README.md
+++ b/README.md
@@ -19,6 +19,15 @@ The module of this project is built to be compatible with Alfresco Content Servi
 
 **Note**: Alfresco Enterprise Edition already ships with custom cache implementations that are tied directly to its Enterprise clustering feature. Though we have managed to install aldica and use its cache provider on a trial version of Alfresco Enterprise Edition with only a minor bit of tinkering, we cannot generally support Alfresco Enterprise Edition. Its code is not open to us and may change without notice / disclosure in any service pack or hotfix release, so we cannot safely adapt aldica to work without risk to Enterprise features in the range of versions of Alfresco Content Services that we would like to support. Furthermore, Enterprise customers choosing to install aldica would likely be left in a system state that Alfresco Support could rightfully refuse to support when issues are filed for problems remotely related to caching. As an open source project, we cannot take on the warranty / liability of such systems either.
 
+### Known Issue(s) / Limitation(s) with Alfresco 5.2
+
+Due to an issue with transactional resource and cleanup handling (e.g. [SPR-15194](https://jira.spring.io/browse/SPR-15194)) fixed only with an upgraded Spring library in Alfresco 6.x, parts of aldica's serialisation improvements to reduce the memory footprint of cached values cannot be used on Alfresco 5.2. These must be disabled by setting the following Repository-tier global properties:
+
+```
+aldica.core.binary.optimisation.useIdsWhenReasonable=false
+aldica.core.binary.optimisation.useIdsWhenPossible=false
+```
+
 ## Published Release / SNAPSHOT Artifacts
 
 All artifacts of released versions of this module will be published on Maven Central, using the group ID _org.aldica_ . SNAPSHOT versions of artifacts will be published on the [Sonatype Open Source Software Repository Hosting](https://oss.sonatype.org) service. In order to reference SNAPSHOT versions in Maven-based projects, an appropriate repository section has to be added to the project's POM file:

